R/init.R

```diff
@@ -141,13 +141,13 @@ init <- function(data,
   join_keys <- data$get_join_keys()
   resolve_modules_datanames <- function(modules) {
     if (inherits(modules, "teal_modules")) {
-      modules$children <- sapply(modules$children, resolve_modules_datanames, simplify = FALSE)
+      modules$children <- lapply(modules$children, resolve_modules_datanames)
```

Comment:
Names are dropped here, is this okey? Elements of `teal_modules` should be named